vizro-ai/src/vizro_ai/dashboard/utils.py

```diff
@@ -0,0 +1,74 @@
+"""Helper Functions For Vizro AI dashboard."""
+
+from dataclasses import dataclass, field
+from typing import Any, Dict
+
+import pandas as pd
+import tqdm.std as tsd
+import vizro.models as vm
+
+
+@dataclass
+class MetadataContent:
+    """Dataclass containing metadata content for a dataframe."""
+
+    df_schema: Dict[str, str]
+    df: pd.DataFrame
+    df_sample: pd.DataFrame
+
+
+@dataclass
+class DfMetadata:
```

Comment:
To be discussed. Do we need this class at all? Instead of just maintaining a variable `all_metadata` or similar?